packages/cashier/src/stores/__tests__/payment-agent-transfer-store.spec.ts

```diff
@@ -1,10 +1,23 @@
 import PaymentAgentTransferStore from '../payment-agent-transfer-store';
 import { routes } from '@deriv/shared';
 import { configure } from 'mobx';
+import { TStores, mockStore } from '@deriv/stores';
```

Comment:
```suggestion
import { mockStore } from '@deriv/stores';
```
@rostislav-deriv Use `TRootStore` instead 🙏🏻